Scripts/generate_release_zip.sh

```diff
@@ -79,11 +82,6 @@ else
   XCPRETTY="xcpretty"
 fi
 
-# Clean DerivedData
```

Comment:
why remove this? is this under the assumption, Jenkins clears it whenever we ran this script. if so, we might need to run this outside Jenkins too and no harm in clearing it again?

Reply:
The idea to allow jenkins to run test-only & build-only in parallel.  having "rm DerivedData" will cause failure since either job might have written files in it.  I have a remove step in jenkins job to clean up DerivedData and /private/tmp after the build is done so that the next build will start with a clean env.